src/ontology/mondo-edit.obo

```diff
@@ -337028,21 +337028,31 @@ id: MONDO:0014597
 name: immunodeficiency 39
 def: "Any primary immunodeficiency disease in which the cause of the disease is a mutation in the IRF7 gene." [MONDO:patterns/disease_series_by_gene]
 subset: gard_rare {source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disorder {source="Orphanet:574918"}
+subset: orphanet_rare {source="Orphanet:574918"}
 subset: rare
 synonym: "IMD39" RELATED ABBREVIATION [MONDO:Lexical, OMIM:616345]
 synonym: "immunodeficiency 39" EXACT [MONDO:Lexical, OMIM:616345]
 synonym: "immunodeficiency type 39" EXACT [MONDORULE:2, OMIM:616345]
 synonym: "IRF7 primary immunodeficiency disease" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "predisposition to severe viral infection due to IRF7 deficiency" EXACT [MONDO:0035547]
 synonym: "primary immunodeficiency disease caused by mutation in IRF7" EXACT [MONDO:design_pattern]
 xref: DOID:0111969 {source="MONDO:equivalentTo"}
+xref: ICD10CM:D84.8 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:574918"}
+xref: MEDGEN:1830120 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MEDGEN:904167 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
```

Comment:
should one of these be preferredExternal?

Reply:
@nicolevasilevsky All the MEDGEN and UMLS cross-references are maintained by MedGen. Therefore anything we do with these x-refs will be automatically updated (and probably lost) during the next refresh. Also, they will be the one deciding what to do (e.g merge their own record). 
For these reasons, I did not add a "preferredExternal".